Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM php:7.3
+
+COPY composer.json /app/
+COPY Classes /app/Classes
+
+RUN useradd -ms /bin/bash application
+
+RUN apt-get update && \
+  apt-get install -y sudo vim nano less git zip unzip && \

```

Comment:
i dont think its necessary to add sudo, vim, nano, less, zip and unzip. it is nowhere needed